storage/go110.go

```diff
@@ -46,6 +46,8 @@ func shouldRetry(err error) bool {
 		return false
 	case interface{ Temporary() bool }:
```

Comment:
We also need to modify this case to allow a wrapped temporary error to be retried. Should match https://github.com/googleapis/google-api-go-client/blob/master/internal/gensupport/resumable.go#L248 .

Reply:
thanks @tritone, would you like me to break the switch case into if conditions instead?